src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -186,6 +207,21 @@ public String getOrgId(OAuthInfo oAuthInfo) throws ConnectionException {
     return partnerConnection.getUserInfo().getOrganizationId();
   }
 
+  @Nullable
+  public Long getInitialRetryDuration() {
+    return initialRetryDuration == null ? 5L : initialRetryDuration;
```

Comment:
create variables for these

Reply:
Added variables for all the constants used.